CompanyEmployees/MappingProfile.cs

```diff
@@ -14,6 +14,10 @@ public MappingProfile()
 
             CreateMap<CompanyForCreationDto, Company>();
             CreateMap<CompanyForUpdateDto, Company>();
+
+            // Attendance mappings
```

Comment:
Consider grouping all attendance-related mappings together in a dedicated region or at the end of similar mapping blocks to improve readability and maintainability as the number of mappings grows.

**Suggested change**
```suggestion
// Attendance mappings
CreateMap<AttendanceForCreationDto, Attendance>();
CreateMap<Attendance, AttendanceDto>();
```